pkg/reconciler/events/cloudevent/cloudevent.go

```diff
@@ -88,18 +89,18 @@ type CEClient interface {
 // TektonCloudEventData type is used to marshal and unmarshal the payload of
 // a Tekton cloud event. It can include a TaskRun or a PipelineRun
 type TektonCloudEventData struct {
-	TaskRun     *v1beta1.TaskRun     `json:"taskRun,omitempty"`
-	PipelineRun *v1beta1.PipelineRun `json:"pipelineRun,omitempty"`
-	CustomRun   *v1beta1.CustomRun   `json:"customRun,omitempty"`
+	TaskRun     *v1.TaskRun        `json:"taskRun,omitempty"`
+	PipelineRun *v1.PipelineRun    `json:"pipelineRun,omitempty"`
+	CustomRun   *v1beta1.CustomRun `json:"customRun,omitempty"`
 }
```

Comment:
This is a change in user interface, so it should be documented in the release notes with an action required.
Users that define their `TaskRuns` and `PipelineRuns` using the `v1beta1` API, will now see CloudEvents with `v1` resources in the payload. 

While this makes sense, and we should definitely do it, it is not strictly required to be in the same PR as the change of storage. Again, I'm not suggesting we move it to a separate PR now, but we should definitely call this out and in future try to keep such changes separated.

Reply:
Thanks Andrea, I updated it in the release note!